python/pyspark/statcounter.py

```diff
@@ -19,6 +19,11 @@
 
 import copy
 import math
+from typing import cast, Any, Dict, Iterable, Optional, Union
+
+maximum: Any
+minimum: Any
+sqrt: Any
```

Comment:
can we avoid this?

Reply:
updated, https://github.com/apache/spark/pull/34435#discussion_r743501203. Thanks!